recipes/souporcell/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -euo
+
+mkdir -p ${PREFIX}/bin
+
+export CFLAGS="${CFLAGS} -fcommon"
+export CXXFLAGS="${CFLAGS} -fcommon -I${PREFIX}/include"
+
+# Include scripts
+cp -f stan_consensus.pickle $PREFIX/bin/stan_consensus.pickle
+cp -f *py $PREFIX/bin
+chmod +rx $PREFIX/bin/souporcell_pipeline.py
+
+# Scripts expect the binary located in the following folder
+mkdir -p $PREFIX/bin/souporcell/target/release
+mkdir -p $PREFIX/bin/troublet/target/release
+
+# Build statically linked binary with Rust
+RUST_BACKTRACE=1
+cargo install --verbose --root "${PREFIX}/bin/souporcell/target/release/" --path souporcell
+cargo install --verbose --root "${PREFIX}/bin/troublet/target/release/" --path troublet
```

Comment:
I think the path is still wrong in the end. Files ultimately end up at `${PREFIX}/bin/souporcell/target/release/bin/`. Might be better to build in place then `cp` to where it needs to go.

Reply:
Using `cargo build --release` ends up with a triplet, but we can use `install` after that (I think).